### DISCUSSION TITLE
debian packages installation

### DISCUSSION THREAD

**Discussion by maaathieu-x** [magma/magma#14022] [Q&A]
Hello,

1.8.0 release notes refers to third party debian packages. 
Indeed, I can find debian packages here:
https://docker.artifactory.magmacore.org/artifactory/debian/

Has somebody already installed the orchestrator with this packages ? I can't find any documentation. 
Unless they only concern the AGW ?

Thanks,

Mathieu

**Comment by sebathomas** [comment]
The debian packages are only for installing the AGW. The orc8r deployment uses helm, see https://docs.magmacore.org/docs/orc8r/deploy_install.
